src/content/developers/docs/scaling/index.md

```diff
@@ -10,6 +10,8 @@ sidebarDepth: 3
 
 As the number of people using Ethereum has grown, the blockchain has reached certain capacity limitations. This has driven up the cost of using the network, creating the need for "scaling solutions." There are multiple solutions being researched, tested and implemented that take different approaches to achieve similar goals.
 
+The defenition of Ethereum scalability is the number of transactions that the blockchain can processes per second. It is also referred to as Transactions per second (TPS) or Throughput.
```

Comment:
I kinda feel like the next paragraph already covers this? Not sure we'd want to include this twice. Do you see something missing from the following paragraph?